src/audio/Kconfig

```diff
@@ -134,6 +134,27 @@ config CORDIC_FIXED
 	  Start with the angle in the range [-2*pi, 2*pi) and output has range in
 	  [-1.0 to 1.0]
 
+config POWER_FIXED
```

Comment:
If #5013 gets merged before this these kconfig menu entries should go to src/math/Kconfig instead.

Reply:
OK.